### PR TITLE
tweak ButtonText, added red color, add cancel modal

### PR DESCRIPTION
**Pull request by davideastmond**
- Add a red color to the style theme
- add a cancel button on the submission form, and functionality to alert user when closing browser tab
- add a confirmation modal when user clicks the cancel button
image
image

### CODE REVIEW

File: src/app/globals.css [hunk 1]
```diff
@@ -1,10 +1,3 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
```
Comment (n3ndor) [inline src/app/globals.css:3]: yes, I like it when the globals.css is empty. Go Tailwind powaaa !

File: src/components/buttonText.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 interface ButtonTextProps {
   text: string;
-  color: "Green" | "Yellow";
+  color: "Green" | "Yellow" | "Red";
   name?: string;
   disabled?: boolean;
+  paddingX?: 2 | 4;
```
Comment (n3ndor) [inline src/components/buttonText.tsx:6]: do we really need to fix only this two options ? Having the possibility to add a className would let us to set flexibly which padding we ever want.

File: src/components/submission-form/Submission-form.tsx [hunk 3]
```diff
@@ -46,6 +46,7 @@ export function SubmissionForm() {
   const { data: session, status } = useSession();
   const [isBusy, setIsBusy] = useState(false);
   const [isSubmitted, setIsSubmitted] = useState(false);
+  const [cancelModalOpen, setCancelModalOpen] = useState(false);
```
Comment (n3ndor) [inline src/components/submission-form/Submission-form.tsx]: little complicated naming convention. I prefer to use openModal - setOpenModal - it's simple and you don't need to combine cancel and open in one name

**Review by n3ndor — APPROVED** [review]

File: src/components/buttonText.tsx (hunk 2 above)
Comment (davideastmond) [reply src/components/buttonText.tsx:6]: I'm open to expanding this. I was following the pattern of restricting the values. If things get out of control in the future, let's expand it to injecting classnames?

File: src/components/submission-form/Submission-form.tsx (hunk 3 above)
Comment (davideastmond) [reply src/components/submission-form/Submission-form.tsx]: Updated

**Review by n3ndor — APPROVED** [review]

**Pull request merged**